### ISSUE TITLE
Some Australian Content Ratings not getting parsed correctly

### ISSUE THREAD

**Issue by purevertigo** [gboudreau/XBMCnfoMoviesImporter.bundle#122]
Australian Ratings -

MA15+ 
R18+ 
X18+

All get parsed without the trailing '+'

Can someone awesome please help fix?

thanks

**Comment by Squazz** [comment]
Is this still a problem?
I just looked at that section of the python yesterday, and it seems that it should handle Australian Ratings, there even are special code just for Australian ratings.

If it's still a problem with the newest version of the plugin, try to provide a NFO that shows this behavior. And make sure you have chosen to use Australia in the settings for the agent: "Country (used for content rating)"

**Comment by purevertigo** [comment]
Yes, this is still a problem. the + at the end does not get parsed through to Plex. for example MA15+ gets scraped as MA15
